tests/test_checkpoint.py

```diff
@@ -241,7 +241,7 @@ def _compare_configs(config_ref, config_test):
 @pytest.mark.depends(on=["test_converted_distributed"])
 def test_load_pretrained_distributed_checkpoint():
     config = TEST_ARCHITECTURE_CONFIG_CLS.from_dict(
-        yaml.safe_load((_CKPT_PATH / ".." / ".." / "config.yaml").open("r")), strict=False
+        yaml.safe_load((_CKPT_PATH / ".." / ".." / "config.yaml").open("r"))["model"]["base_model"], strict=False
```

Comment:
I don't understand this one. Wasn't it working before? I'd have caught it in tests...

Reply:
The config comparison was broken I think (fixed in https://github.com/ServiceNow/Fast-LLM/pull/55/commits/fa6440e1cf2fc9d0aa4d050e7a17bdcc6e804619), which is why this one wasn't caught